test/parallel/test-child-process-stdio.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
-var options = {stdio: ['pipe']};
-var child = common.spawnPwd(options);
+let options = {stdio: ['pipe']};
+let child = common.spawnPwd(options);
 
 assert.notEqual(child.stdout, null);
 assert.notEqual(child.stderr, null);
```

Comment:
@addaleax If i understood that correctly: 

Do you mean that i should replace equal with strictEqual and notEqual with notStrictEqual ? 

In that case will there be any inconsistencies because of the fact that the assertion in this file are against null and: 
```undefined === null // equals false ``` 
``` undefined !== null // equals true ```
 
For example: 
```assert.notStrictEqual(undefined, null) // this will pass ```
```assert.strictEqual(undefined, null) // this will not pass ```